src/LandIce/evaluators/hydrology/LandIce_EffectivePressure.hpp

```diff
@@ -56,30 +54,25 @@ class EffectivePressure : public PHX::EvaluatorWithBaseImpl<Traits>
   PHX::MDField<HydroScalarT>  N;
 
   int numPts;
+  unsigned int worksetSize;
 
   bool eval_on_side;
   std::string sideSetName; // Needed if OnSide=true
 
   PHX::MDField<const ScalarT,Dim> alphaParam;
   ScalarT printedAlpha;
-
-  bool useCollapsedSidesets;
   
   Albany::LocalSideSetInfo sideSet;
 
 public:
 
   typedef Kokkos::View<int***, PHX::Device>::execution_space ExecutionSpace;
-  struct Surrogate_Tag{};
-  struct NonSurrogate_Tag{};
+  struct EffectivePressure_Tag{};
```

Comment:
Not really important, but since there's only one tag, do we still need tags?

Reply:
This is just a style choice on my part. If at some point any of these evaluators are expanded to need multiple kernels, the tag and policy stuff would be there as an example.